.github/workflows/test-env-deploy.yml

```diff
@@ -165,12 +164,39 @@ jobs:
               for(let i=1; i<=numberOfContainers[1]; i++){
                 containers.push(i)
               }
+            echo ${{Array.from(containers.keys())}}
             return {"containers": containers}
 
       - name: echo-tags
         run: |
           echo ${{steps.get_tags.outputs.result}}
 
+  testmo-report-preparation:
+    needs: prepare-tests
+    runs-on: ubuntu-latest
+    outputs:
+      testmo-run-id: ${{ steps.run-tests.outputs.testmo-run-id }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 19
+          cache: npm
+      - run: npm ci
+      - run: |
+          npx testmo automation:run:create \
```

Comment:
Aren't you missing the `@testmo/testmo-cli` in `.github/workflows/package.json`?

Reply:
Yeah :) Fixed